deploy/agent-mcq.setup

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+
+install_requires = [
+    "pyzmq>=15.3.0",
+    "msgpack >= 1.0.0",
+    "docker>=2.5.0",
```

Comment:
```suggestion

```

Reply:
Removed in https://github.com/UCL-INGI/INGInious/pull/882/commits/13e381ef3da8a4cc068bcf455fc441f3f4a671da.